internal/store/api_keys.go

```diff
@@ -45,6 +45,10 @@ type OrganizationAPIKeyCreator interface {
 	CreateOrganizationAPIKey(string, *atlas.APIKeyInput) (*atlas.APIKey, error)
 }
 
+type OrganizationAPIKeyDeleter interface {
+	DeleteOrganizationAPIKeys(string, string) error
```

Comment:
[nit] this should be singular
```suggestion
	DeleteOrganizationAPIKey(string, string) error
```
(Don't forget to update the implementation